react-toggle-switch/my-app/src/ToggleSwitch.tsx

```diff
@@ -0,0 +1,16 @@
+import './ToggleSwitch.css';
+import { useState } from 'react';
+
+export function ToggleSwitch() {
+  const [tSwitch, setTSwitch] = useState(false);
```

Comment:
This name doesn't tell me what the state means. If `tSwitch` is true, is the switch on or off?